### PR TITLE
feat(block-content-website): anonymous poll block voting & embed cleanup

### PR DESCRIPTION
**Pull request by Itrulia**
1. Moved embeds into their own folders like the other blocks
2. Added the embeds to blocks/article/page stories
3. Fixed EmbedBlock to not use old implemention
4. Fixed TikTok block

### CODE REVIEW

File: libs/api/src/lib/graphql/poll/poll.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export const GraphQLPollVote = new GraphQLObjectType<PollVote, Context>({
   name: 'PollVote',
   fields: {
     answerId: {type: GraphQLNonNull(GraphQLString)},
+    pollId: {type: GraphQLNonNull(GraphQLID)},
```
Comment (Itrulia) [inline libs/api/src/lib/graphql/poll/poll.ts:99]: Makes it easier to implement anonymous voting

File: libs/block-content/website/src/lib/embed/embed-block.tsx
Comment (Itrulia) [inline libs/block-content/website/src/lib/embed/embed-block.tsx:1]: CC: @antkiewiczk  cleaned this up to not use custom style processor and stuff

File: libs/block-content/website/src/lib/tik-tok/tik-tok-video-block.tsx
Comment (Itrulia) [inline libs/block-content/website/src/lib/tik-tok/tik-tok-video-block.tsx:1]: CC: @antkiewiczk fixed height of tiktok player being too big and CSS specifity to 1.

From PR Comment: https://github.com/wepublish/wepublish/pull/1002#discussion_r1279252546

**Review by antkiewiczk — APPROVED** [review]

File: libs/block-content/website/src/lib/embed/embed-block.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+export function EmbedBlock({
+  url,
+  title,
+  width,
+  height,
+  styleCustom,
+  sandbox,
+  className
+}: BuilderEmbedBlockProps) {
+  const ratio = width && height ? width / height : 0
```
[the remark this answers is not in the document]
Comment (Itrulia) [reply libs/block-content/website/src/lib/embed/embed-block.tsx:25]: 1 is the fallback further down when height or width isnt set. Tbh could also be set here, makes more sense

File: libs/block-content/website/src/lib/embed/embed-block.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  const styleCustomCss = useMemo(
+    () => css`
+      width: 100%;
```
Comment (Itrulia) [inline libs/block-content/website/src/lib/embed/embed-block.tsx:29]: reminder for myself: move this up to the styled

**Pull request merged**